### ISSUE TITLE
[Docs] Improve cong server documentation

### ISSUE THREAD

**Issue by mtdvlpr** [sircharlo/meeting-media-manager#1320]
I tested the dynamics of the WebDAV server and found it a bit "confusing" for general users to sign up and use. Why am I saying this? I'm testing Koofr's service and I've done the following and it works fine:
1. Installed on PC
2. I created the folders on the PC inside Koofr
3. Similar to Google Drive, syncs well
4. On M3, I put the folder that I'm going to use on KH's PC

https://app.koofr.net/
### On-line [Site]
![image](https://user-images.githubusercontent.com/90984283/215240608-c8fee1d0-a267-4aa4-92b0-b7a51a219f89.png)

### PC KH
![image](https://user-images.githubusercontent.com/90984283/215240765-c5cd1284-7577-43fb-ac0c-f6a92a5c87a7.png)

**_I know that wasn't the idea, but when I download the media here, and the team opens it in KH, the media syncs similar to Google Drive that I mentioned in another post._**

_Originally posted by @e-Feitosa in https://github.com/sircharlo/meeting-media-manager/issues/1264#issuecomment-1407278387_

**Comment by mtdvlpr** [comment]
Multiple users have been confused with the way the cong server works. This has to do with their previous experience using Google Drive or OneDrive. The misconception is either that the cong server is supposed to contain all media (including standard JW meeting media) or that the local output folder is supposed to be a local synced folder of the cong server.

**Comment by Chipperchap** [comment]
Not sure if this is accurate but how I understand it...?

![M3 Media Explained](https://user-images.githubusercontent.com/35364657/215298407-b61d83cd-4fdc-40af-a333-767697e68517.png)

**Comment by mtdvlpr** [comment]
@Chipperchap spot on!

**Comment by e-Feitosa** [comment]
That's what I understood too, but analyzing it well, I came to understand that ONLY the additional (extra) media that will be added in WebDAV and in the Kingdom Hall, the JW media are downloaded locally on the PC and the cloud media (koofr, 4share etc) are downloaded to the congregation's server on the local PC (Kingdom Hall). As I understand it, you don't need to download the JW media on the server. 

**In short**: Server media are additional media that are not in the jw.

**Comment by mtdvlpr** [comment]
@e-Feitosa, exactly! And to be more precise: server media are additional media that are not included in the standard meeting. Public speaker song, governing body update, etc are also additional media that are stored in the congregation server.

**Comment by e-Feitosa** [comment]
I'm still having some difficulty here in synchronizing. WebDAV server Koofr connected on M3 (OK!). Then I added the missing media, on the M3, I see the media only in the Congregation Server section in settings, but the media are not being downloaded to the M3. I believe it's something simple to solve, still testing here.

Koofr + M3 (Config. Server)
![image](https://user-images.githubusercontent.com/90984283/215352654-c121ad03-e230-4ba3-876e-d7f2be64bb4c.png)

M3
![image](https://user-images.githubusercontent.com/90984283/215352678-f29a4ed0-5940-4ef8-a25b-54bc699b5871.png)

M3
![image](https://user-images.githubusercontent.com/90984283/215352722-9d958ca6-cc42-4ce6-a13f-c2ba36bce567.png)

**Comment by mtdvlpr** [comment]
@e-Feitosa, how did you add the media? Did you upload them to Koofr yourself? They should be in a media/[date] folder. When added by opening a date in M3, they should be placed in the correct location.

**Comment by mtdvlpr** [comment]
Try clicking on the Sunday and uploading the images through that page.

**Comment by e-Feitosa** [comment]
> @e-Feitosa, como você adicionou a mídia? Você mesmo os enviou para o Koofr? Eles devem estar em uma pasta de mídia/[data]. Quando adicionados abrindo uma data no M3, eles devem ser colocados no local correto.

Sim, abra na web o site do koofr e adicionei as mídias, depois das suas respostas, criei uma pasta mídia / [data (igual no M3) ] e adicionei as mídias. Ainda não tive êxito para baixar no M3, eu vejo os arquivos no M3 que adicionadoi no WebDAV Koofr, somente não baixar as mídias para o M3 [é só questão de configuração mesmo! 😁].

**Comment by mtdvlpr** [comment]
@e-Feitosa, you shouldn't create those folders yourself. M3 makes them for you when you add the media the proper way (through the manage media screen in M3). See [documentation](https://sircharlo.github.io/meeting-media-manager/en/#manage-media)

**Comment by e-Feitosa** [comment]
Working perfectly.
I tested koofr and 4Shared (both work well), but I chose to use koofr because it generates an application password for the M3, increasing the security of the application.

In short, it's a lot simpler than I thought:

1. Open the website [https://app.koofr.net/login] and create an account

2. Then access preferences and passwords and use the "App passwords" option

3. Name the password (type M3DAV) and generate the APP password

4. Ready, in M3, enter the e-mail and password generated and click on the globe

Where was the problem:
The idea persisted that I had to create the same folders as in Google Drive, in this case, the JW media downloads in a folder on the LOCAL PC and only the additional media will be included in the WebDAV (Congregation Server)
__
**Observation:**
I believe it is normal, both WebDAV services take less than 1 minute to synchronize the media that are added.

**PC HOME**
⬇️
![image](https://user-images.githubusercontent.com/90984283/215374066-7e440db8-df5e-4925-ac7f-208732230c03.png)

**This date setting must be done the same on all PCs that will synchronize:**
⬇️
![image](https://user-images.githubusercontent.com/90984283/215374339-7bc977a4-7340-457a-b6dc-f254cf981e20.png)

PC HOME 💻 PC KH | **server configuration**
⬇️ [Por exemplo: koofr]
![image](https://user-images.githubusercontent.com/90984283/215374536-3ea9942b-c8ad-4c47-8543-65a43aeee526.png)

**Comment by mtdvlpr** [comment]
@e-Feitosa, great! Small tip for you: you can lock the date format (among other things) so that everyone automatically has the correct one. Use the blue button in the congregation sync setup settings.

**Comment by e-Feitosa** [comment]
> @e-Feitosa, great! Small tip for you: you can lock the date format (among other things) so that everyone automatically has the correct one. Use the blue icon button in the congregation sync setup settings.

**_Very good! Thanks again_**
